io.c

```diff
@@ -2198,7 +2199,7 @@ rb_io_flush(VALUE io)
  *    tell -> integer
  *
  *  Returns the current position (in bytes) in +self+
- *  (see {Position}[rdoc-ref:IO@Position]):
+ *  (see {Position}[rdoc-ref:doc/io_streams.rdoc@Position]):
```

Comment:
This is a broken link.
```suggestion
 *  (see {Position}[rdoc-ref:io_streams.rdoc@Position]):
```

Reply:
> This is a broken link, and all others.

Not broken on my machine.  I don't have make, so I run `rdoc --op html`, and have no breaks.  @peterzhu2118, how about you?  Or anyone else?

Reply:
This isn't broken for me either, but the suggested change from @nobu also works for me.

Reply:
Fixed (37 places).

Reply:
> This is a broken link, and all others.

These are fixed and verified.